src/tools/schema2kotlin.ts

```diff
@@ -83,19 +96,25 @@ import arcs.core.storage.api.toPrimitiveValue`}
       const handleConcreteType = connection.type.isCollectionType() ? 'Collection' : 'Singleton';
       let handleInterfaceType: string;
       if (this.opts.wasm) {
-        handleInterfaceType = `Wasm${handleConcreteType}Impl<${entityType}>`;
+        handleInterfaceType = this.getType(`${handleConcreteType}Impl<${entityType}>`);
```

Comment:
I think the term "type" is a bit overloaded --> is there a better name for Wasm vs Jvm mode?

Reply:
* I'm referring to the function `getType`

Reply:
Changing name to `prefixTypeForRuntime`.